testsuite/src/it/java/io/quarkus/ts/startstop/LiveReloadTest.java

```diff
@@ -0,0 +1,215 @@
+package io.quarkus.ts.startstop;
+
+import static io.quarkus.ts.startstop.utils.Commands.cleanTarget;
+import static io.quarkus.ts.startstop.utils.Commands.getBaseDir;
+import static io.quarkus.ts.startstop.utils.Commands.getRunCommand;
+import static io.quarkus.ts.startstop.utils.Commands.processStopper;
+import static io.quarkus.ts.startstop.utils.Commands.runCommand;
+import static io.quarkus.ts.startstop.utils.Logs.appendln;
+import static io.quarkus.ts.startstop.utils.Logs.appendlnSection;
+import static io.quarkus.ts.startstop.utils.Logs.archiveLog;
+import static io.quarkus.ts.startstop.utils.Logs.writeReport;
+
+import java.io.File;
+import java.io.IOException;
+import java.lang.reflect.Method;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Date;
+import java.util.List;
+
+import org.apache.commons.io.FileUtils;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.TestInfo;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.EnumSource;
+
+import io.quarkus.ts.startstop.utils.Apps;
+import io.quarkus.ts.startstop.utils.HttpUtils;
+import io.quarkus.ts.startstop.utils.MvnCmds;
+
+@Tag("instrumentation")
```

Comment:
Do we want a new tag, which would have to be added to all relevant jenkins jobs?
Or use existing tag?

Reply:
IMO we don't need it